src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -0,0 +1,183 @@
+            "CredSSP",
+            "Digest",
+            "Kerberos")] // can be used with explicit or implicit credential
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public string WsmanAuthentication { get; set; } = "Default";
+
+        /// <summary>
+        /// The following is the definition of the input parameter "ComputerName".
+        /// Value of the address requested. The form of the value can be either the
+        /// computer name ("wxyz1234"), IPv4 address ("192.168.177.124"), or IPv6
+        /// address ("2010:836B:4179::836B:4179").
+        /// </summary>
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        [Alias("CN", "__SERVER", "Server", "IPAddress")]
+        public string[] ComputerName { get; set; } = new string[] { "." };
+
+        /// <summary>
+        /// The following is the definition of the input parameter "Credential".
+        /// Specifies a user account that has permission to perform this action. Type a
+        /// user-name, such as "User01" or "Domain01\User01", or enter a PSCredential
+        /// object, such as one from the Get-Credential cmdlet.
+        /// </summary>
+        [Parameter(Position = 1)]
+        [ValidateNotNullOrEmpty]
+        [Credential]
+        public PSCredential Credential { get; set; }
+
+        /// <summary>
+        /// Force the operation to take place if possible.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Force { get; set; } = false;
+
+        #endregion "parameters"
+
+        #region "IDisposable Members"
+
+        /// <summary>
+        /// Dispose Method.
+        /// </summary>
+        public void Dispose()
+        {
+            try
+            {
+                _process.Dispose();
+            }
+            catch (ObjectDisposedException) { }
+        }
+
+        #endregion "IDisposable Members"
+
+        #region "Overrides"
+
+        /// <summary>
+        /// BeginProcessing.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            if (ComputerName.Length > 1) {
+                ErrorRecord error = new ErrorRecord(
+                        new InvalidOperationException(ComputerResources.LocalhostOnly), "InvalidNewName",
+                        ErrorCategory.InvalidArgument, ComputerName);
+                WriteError(error);
+            }
+            var computer = ComputerName[0];
+            if ((computer.Equals("localhost", StringComparison.OrdinalIgnoreCase)) || (computer.Equals(".", StringComparison.OrdinalIgnoreCase)))
+            {
+                doShutdown();
+                return;
+            }
+            WriteError(new ErrorRecord(
```

Comment:
Missed this path, where we can use the same ComputerResources.LocalhostOnly message string.

Reply:
done.